pkg/oc/admin/project/new_project.go

```diff
@@ -127,7 +128,12 @@ func (o *NewProjectOptions) Run(useNodeSelector bool) error {
 		}
 	}
 
-	for _, binding := range bootstrappolicy.GetBootstrapServiceAccountProjectRoleBindings(o.ProjectName) {
+	for _, rbacBinding := range bootstrappolicy.GetBootstrapServiceAccountProjectRoleBindings(o.ProjectName) {
+		binding, err := authzregutil.RoleBindingFromRBAC(&rbacBinding)
+		if err != nil {
+			errs = append(errs, err)
```

Comment:
`fmt.Printf("Could not convert Role Binding %s in the %q namespace: %v\n", rbacBinding.Name, o.ProjectName, err)`